src/java/fr/paris/lutece/portal/resources/rbac_messages.properties

```diff
@@ -17,7 +17,7 @@ pageTitle.selectPermissions=Permission selection
 ################################################################################
 # Messages
 message.confirmDeleteRole=Do you really want to delete this role ?
-message.confirmDeleteControl=Do you really want to delete this control ?
+message.confirmDeleteControl=Do you really want to delete the control "{1}" for the resource "{1}" of type "{2}" for role "{0}" ?

```

Comment:
hi
Could there be an error in the indices?
it should be : 
for the resource "{2}" of type "{3}" ?

Reply:
yes you are right